pkg/common/version/version.go

```diff
@@ -8,7 +8,7 @@ const (
 	// IMPORTANT: When updating, make sure to reconcile the versions list that
 	// is part of the upgrade integration test. See
 	// test/integration/suites/upgrade/README.md for details.
```

Comment:
This comment is a reminder that we also need to add 1.0.0 to the versions.txt in the upgrade test suite which we normally do as part of the same PR that bumps the version.

Reply:
Lol whoops, I flew right past that. Fixed